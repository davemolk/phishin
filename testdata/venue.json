{"success":true,"total_entries":1,"total_pages":1,"page":1,"data":{"id":11,"slug":"the-academy","name":"The Academy","other_names":[],"latitude":40.783515,"longitude":-73.958766,"location":"New York, NY","city":"New York","state":"NY","country":"USA","shows_count":1,"show_dates":["1991-07-15"],"show_ids":[472],"updated_at":"2013-03-24T03:17:31Z"}}