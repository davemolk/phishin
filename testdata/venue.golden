Venue:       Location:     Show Count:
The Academy  New York, NY  1

Show Dates
1991-07-15
